src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -781,7 +781,9 @@ internal async Task<JToken> RuntimeGetPropertiesInternal(SessionId id, DotnetObj
 
                 }
             }
-            catch (Exception) {
+            catch (Exception ex)
+            {
+                logger.LogDebug($"Error on getting properties of object {objectId} with scheme {objectId.Scheme}. {ex.Message}.");
```

Comment:
For debugging, the full stack trace would be more useful.
```suggestion
                logger.LogDebug($"Error on getting properties of object {objectId} with scheme {objectId.Scheme}. {ex}.");
```